src/lib.rs

```diff
@@ -17,16 +11,16 @@ pub mod alloc;
 pub mod error;
 pub mod native_types;
 pub mod raw;
+use strum_macros::AsRefStr;
```

Comment:
Nice, but it'd be even nicer to put all the `use` statements together at the top and not mixed with `mod` statements.

Reply:
Whoops. I was counting on the formatter to sort it out.